sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormTrainingAsyncClientJavaDocCodeSnippets.java

```diff
@@ -106,72 +108,72 @@ public void getCustomModelWithResponse() {
         formTrainingAsyncClient.getAccountPropertiesWithResponse().subscribe(response -> {
             System.out.printf("Response Status Code: %d.", response.getStatusCode());
             AccountProperties accountProperties = response.getValue();
             System.out.printf("Max number of models that can be trained for this account: %s%n",
-                accountProperties.getLimit());
-            System.out.printf("Current count of trained custom models: %d%n", accountProperties.getCount());
+                accountProperties.getCustomModelLimit());
+            System.out.printf("Current count of trained custom models: %d%n", accountProperties.getCustomModelCount());
         });
-        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getAccountPropertiesWithResponse
+        // END: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.getAccountPropertiesWithResponse
     }
 
     /**
      * Code snippet for {@link FormTrainingAsyncClient#deleteModel}
      */
     public void deleteModel() {
-        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.deleteModel#string
+        // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.deleteModel#string
         String modelId = "{model_id}";
         formTrainingAsyncClient.deleteModel(modelId).subscribe(val ->
             System.out.printf("Model Id: %s is deleted%n", modelId));
-        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.deleteModel#string
+        // END: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.deleteModel#string
     }
 
     /**
-     * Code snippet for {@link FormTrainingAsyncClient#deleteModelWithResponse}
+     * Code snippet for {@link FormTrainingAsyncClient#deleteModelWithResponse(String)}
      */
     public void deleteModelWithResponse() {
-        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.deleteModelWithResponse#string
+        // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.deleteModelWithResponse#string
         String modelId = "{model_id}";
         formTrainingAsyncClient.deleteModelWithResponse(modelId).subscribe(response -> {
             System.out.printf("Response Status Code: %d.", response.getStatusCode());
             System.out.printf("Model Id: %s is deleted%n", modelId);
         });
-        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.deleteModelWithResponse#string
+        // END: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.deleteModelWithResponse#string
     }
 
     /**
-     * Code snippet for {@link FormTrainingAsyncClient#getModelInfos}
+     * Code snippet for {@link FormTrainingAsyncClient#getModelInfos()}
      */
     public void getModelInfos() {
-        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getModelInfos
+        // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.getModelInfos
```

Comment:
maybe you're going to address this in another PR, but from this meeting we also determined to change getModelInfos -> getCustomModels

Reply:
yup, will address some of the name changes in another PR.